scripts/cc-cli/src/utils/account.ts

```diff
@@ -1,21 +1,56 @@
+import { mnemonicValidate } from "@polkadot/util-crypto";
 import { ApiPromise, Keyring } from "creditcoin-js";
-import { OptionValues } from "commander";
-import { readFileSync } from "fs";
+import prompts from "prompts";
 
 export function initKeyringPair(seed: string) {
   const keyring = new Keyring({ type: "sr25519" });
   const pair = keyring.addFromUri(`${seed}`);
   return pair;
 }
 
-export function getSeedFromOptions(options: OptionValues) {
-  if (options.seed) {
-    return options.seed;
-  } else if (options.file) {
-    return readFileSync(options.file).toString();
-  } else {
-    throw new Error("Must specify either mnemonic phrase or file as an option");
+export async function getStashSeedFromEnvOrPrompt() {
+  return await getSeedFromEnvOrPrompt(
+    process.env.CC_STASH_SEED,
+    "Specify a seed phrase for the Stash account"
+  );
+}
+export async function getControllerSeedFromEnvOrPrompt() {
+  return await getSeedFromEnvOrPrompt(
+    process.env.CC_CONTROLLER_SEED,
+    "Specify a seed phrase for the Controller account"
+  );
+}
+
+export async function getSeedFromEnvOrPrompt(
+  envVar?: string | undefined,
+  promptStr?: string | null
+) {
+  if (envVar) {
+    if (mnemonicValidate(envVar)) {
+      return envVar;
+    } else {
+      console.log(
+        "Error: Seed phrase provided in environment variable is invalid."
+      );
+      process.exit(1);
+    }
+  }
+  const seedPromptResult = await prompts([
+    {
+      type: "invisible",
```

Comment:
What are the other types here ? 

I think from UX POV an asterisk would be better b/c it gives you some feedback, at least how many words you've typed in.

Reply:
done